app/assets/javascripts/application.js

```diff
@@ -1,12 +1,6 @@
-/*
- * = require vendor/jquery-1.11.3.min.js
- * = require d3
- * = require jquery-accessibleMegaMenu.js
- * = require utility.js
- * = require components.js
- * = require cookies.js
- * = require drawer.js
- * = require navDropdown.js
- */
-
-
+//= require jquery-accessibleMegaMenu.js
+//= require d3.v3.min
```

Comment:
Ah ok. I was wondering if that was the case because you had said it was working. I figured you may have changed something after that comment